cleanlab/multiannotator.py

```diff
@@ -0,0 +1,829 @@
+import numpy as np
+import pandas as pd
+from typing import List, Union, Tuple
+from cleanlab.rank import get_label_quality_scores
+from cleanlab.internal.util import get_num_classes
+from cleanlab.internal.label_quality_utils import _get_label_quality_scores_with_NA
+from cleanlab.dataset import _get_worst_class
+import warnings
+
+
+def convert_long_to_wide_dataset(
+    labels_multiannotator_long: pd.DataFrame,
+) -> pd.DataFrame:
+    """Converts a wide format dataset to long format which is suitable for passing into
```

Comment:
I think you have it reversed in the description :) should be long to wide

Reply:
Oops good catch!